src/components/Login.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import './Login.css';
+import { fetchUsers } from '../apiCalls'
+
+class Login extends Component {
+  constructor(){
+    super()
+    this.state = {
+      email: '',
+      password: '',
+      errorMsg: '',
+      user: ''
+    }
+  }
+
+  updateForm = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  validateInputs() {
+    const { email, password } = this.state;
+    return !!(email.trim() && password.trim());
+  }
+
+  retrieveUsers = async (event) => {
```

Comment:
Nice first use of async! I still have to study up on exactly how all this syntax fits together.